.doc_gen/metadata/iam_metadata.yaml

```diff
@@ -1161,6 +1269,18 @@ iam_DetachRolePolicy:
   synopsis: detach an &IAM; policy from a role.
   category:
   languages:
+    .NET:
+      versions:
+        - sdk_version: 3
+          github: dotnetv3/IAM
+          sdkguide:
+          excerpts:
+            - description:
+              snippet_tags:
+                - IAM.dotnetv3.DetachRolePolicyExample
+            - description:
+              snippet_tags:
+                - IAM.dotnetv3.DeleteResourcesAsync
```

Comment:
Include only one of these--I'd pick the second because it's the tested one, but it's your call.

Reply:
removed the first